packages/block-editor/src/hooks/position.js

```diff
@@ -283,13 +279,11 @@ export function PositionPanelPure( {
 			options.length > 1 ? (
 				<InspectorControls group="position">
 					<BaseControl
-						className="block-editor-hooks__position-selection"
```

Comment:
Any chance someone was relying on these custom classes?

Reply:
I couldn't find direct usages in Gutenberg, and I also did a quick scan on WP Directory without finding any matches.